unified/build_unified.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -e
+#
+# Copyright (C) 2020 ScyllaDB
+#
+
+#
+# This file is part of Scylla.
+#
+# Scylla is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Scylla is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+if [ $# -eq 0 ]; then
+    echo "usage: $0 [relocatable packages]"
+    exit 1
+fi
+
+rm -rf build/unified
+mkdir -p build/unified
+for pkg in "$@"; do
+    tar -C build/unified -xpf "$pkg"
+done
+for pkg in scylla scylla-python3 scylla-jmx scylla-tools; do
+    if [ ! -d build/unified/$pkg ]; then
```

Comment:
What about separate modes?

We'll want to make this part of ninja, so we can write ninja build/dev/scylla-unified-package.tar.gz, so different modes can't share the same temporary directory.

Reply:
@avikivity the issue on this PR has been fixed, but I think relocatable / .rpm / .deb package build directories also shared between different modes, I suppose we need to fix them too since we support to build it on ninja, correct?